### DISCUSSION TITLE
How to remove scroll glow in Flutter?

### DISCUSSION THREAD

**Discussion by flutterindias** [community/community#28072] [Mobile] [answered]
How to remove scroll glow in Flutter?

**Comment by sthakrishnakumar** [comment, marked as answer]
Hello @flutterindias ,
         You can use BouncingScrollPhysics() in the physics property of the Listview wherever you want to remove the glow.
ListView(
physics: BouncingScrollPhysics(),
)

OR 
you can Wrap the Widget as

NotificationListener<OverscrollIndicatorNotification>(
  onNotification: (overscroll) {
    overscroll.disallowGlow();
  },
  child: ListView(
    //Your code snippets
  ),
),

Hope this would help you

**Reply by flutterindias** [reply]
Thank you so much @sthakrishnakumar
